src/main/java/com/team4/testingsystem/controllers/ContentFilesController.java

```diff
@@ -0,0 +1,48 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.dto.ContentFileRequest;
+import com.team4.testingsystem.entities.ContentFile;
+import com.team4.testingsystem.services.impl.ContentFilesServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/content_files")
+public class ContentFilesController {
+
+    @Autowired
+    private ContentFilesServiceImpl contentFilesService;
```

Comment:
It seems that bean of type `ContentFilesService` should be injected, not the specific implementation.